grafana/resource_dashboard_test.go

```diff
@@ -19,32 +19,46 @@ func TestAccDashboard_basic(t *testing.T) {
 		ProviderFactories: testAccProviderFactories,
 		CheckDestroy:      testAccDashboardCheckDestroy(&dashboard),
 		Steps: []resource.TestStep{
-			// first step creates the resource
 			{
+				// Test resource creation.
 				Config: testAccExample(t, "resources/grafana_dashboard/_acc_basic.tf"),
 				Check: resource.ComposeTestCheckFunc(
 					testAccDashboardCheckExists("grafana_dashboard.test", &dashboard),
 					resource.TestCheckResourceAttr("grafana_dashboard.test", "id", "basic"),
 					resource.TestCheckResourceAttr("grafana_dashboard.test", "uid", "basic"),
-					resource.TestMatchResourceAttr(
-						"grafana_dashboard.test", "config_json", regexp.MustCompile(".*Terraform Acceptance Test.*"),
+					resource.TestCheckResourceAttr(
+						"grafana_dashboard.test", "config_json", "{\"title\":\"Terraform Acceptance Test\",\"uid\":\"basic\"}",
```

Comment:
Small nit: how about avoiding escaping quotes by using backticks:
```suggestion
						"grafana_dashboard.test", "config_json", `{"title":"Terraform Acceptance Test","uid":"basic"}`,
```

Similar to the changes below.